spot-termination-handler/templates/daemonset.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: "{{ template "spotTerminationHandler.fullname" . }}"
+  labels:
+    app: "{{ template "spotTerminationHandler.fullname" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        heritage: "{{ .Release.Service }}"
+        release: "{{ .Release.Name }}"
+        app: {{ template "spotTerminationHandler.fullname" . }}
+        component: "{{ template "spotTerminationHandler.fullname" . }}"
+      annotations:
+        prometheus.io/scrape: "true"
```

Comment:
Should be check `metrics.enabled` before setting this annotation, and use `prometheus.io/path` too